arrow-test/src/main/kotlin/arrow/test/generators/Generators.kt

```diff
@@ -162,4 +163,7 @@ fun <A> genSequenceKW(genA: Gen<A>): Gen<SequenceKW<A>> =
         Gen.create { Gen.list(genA).generate().asSequence().k() }
 
 fun genChars(): Gen<Char> =
-        Gen.oneOf(('A'..'Z') + ('a'..'z') + ('0'..'9') + "!@#$%%^&*()_-~`,<.?/:;}{][±§".toList())
\ No newline at end of file
+        Gen.oneOf(('A'..'Z') + ('a'..'z') + ('0'..'9') + "!@#$%%^&*()_-~`,<.?/:;}{][±§".toList())
+
+fun <A> genSetKW(genA: Gen<A>): Gen<SetKW<A>> =
+        Gen.create { Gen.set(genA).generate().k() }
```

Comment:
The seed resets on every call, messing with entropy. Capture `Gen.set(genA)` as a value outside the scope and you should be fine.